packages/react/src/components/Label/index.js

```diff
@@ -4,17 +4,26 @@ import React from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
 
-const Label = ({ className, htmlFor, ...props }) => (
-  <label className={cx("Label", className)} htmlFor={htmlFor} {...props} />
+import { refShapes } from "../../constants";
+
+const Label = ({ className, forwardedRef, htmlFor, ...props }) => (
+  <label
+    className={cx("Label", className)}
+    htmlFor={htmlFor}
```

Comment:
Can `htmlFor={htmlFor}` be removed since `{...props}` would pass it?

Reply:
Yes and no, because we want it `required` into the propTypes